test/SourceKit/Indexing/index_is_test_candidate.swift

```diff
@@ -13,7 +13,7 @@ struct MyStruct {
   func test_startsWithTest_takesNoParams_returnsVoid_butIsDefinedOnAStruct() {}
 }
 
-private class MyPrivateClass() {
```

Comment:
It doesn't fully compile, but it does seem to parse well enough that the indexing code can do its thing. I don't remember seeing any diagnostics about it, I just happened to notice it looked weird when I was fixing up the Linux expected response.